internal/pubsub/pubsub.go

```diff
@@ -117,15 +99,12 @@ func NewTopic(d driver.Topic) *Topic {
 		}
 
 		callCtx := context.TODO()
-		err := retry.Call(callCtx, gax.Backoff{}, d.IsRetryable, func() error {
+		return retry.Call(callCtx, gax.Backoff{}, d.IsRetryable, func() error {
 			return d.SendBatch(callCtx, dms)
 		})
-		for _, mec := range mecs {
-			mec.errChan <- err
-		}
 	}
-	b := bundler.NewBundler(msgErrChan{}, handler)
-	b.DelayThreshold = time.Millisecond
+	maxHandlers := 1
```

Comment:
We can leave it at 1 if desired. I just wrote a more general package, in case we ever want to add concurrency for performance.

I do suggest we require drivers to be threadsafe, so we can change it to 2 or higher without races. We should probably have a test using `-race` that sets it to something greater than 1.

Reply:
Filed #820